config/routes.rb

```diff
@@ -2,6 +2,10 @@
   resources :users, :only => [:show, :update], :path => "u"
   devise_for :users, :controllers => { :omniauth_callbacks => "users/omniauth_callbacks" }, :skip => [:password]
   
-  resources :projects
+  resources :projects do
```

Comment:
2-space indenting preferred please :-) at least let's be consistent across the project